ThirdPartyAdapters/applovin/applovin/src/main/java/com/google/ads/mediation/applovin/AppLovinWaterfallRewardedRenderer.java

```diff
@@ -97,6 +97,7 @@ public void onInitializeSuccess(@NonNull String sdkKey) {
               incentivizedInterstitial =
                   AppLovinIncentivizedInterstitial.create(zoneId, appLovinSdk);
             }
+            incentivizedInterstitial.setExtraInfo("google_watermark", adConfiguration.getWatermark());
```

Comment:
Not sure if we need this. They call this from their RTB adapter, but clearly it is for waterfall rewarded ads.

Reply:
Should be fine - but I'd double-check in the channel.